src/pymovements/datasets/dataset.py

```diff
@@ -473,7 +475,7 @@ def detect_events(
             eye: str | None = 'auto',
             clear: bool = False,
             verbose: bool = True,
-            **kwargs,
+            **kwargs: int,
```

Comment:
can you use `float | int | str` here? then it's equal to `pymovements.events.EventDetectionCallable`

Reply:
I then get:

```console
mypy.....................................................................Failed
- hook id: mypy
- exit code: 1

src/pymovements/datasets/dataset.py:554: error: Argument 4 to "__call__" of "EventDetectionCallable" has incompatible type "**Dict[str, Union[float, int, str]]"; expected "int"  [arg-type]
Found 1 error in 1 file (checked 1 source file)
```

Reply:
that's very strange, but then lets keep it that way for now and fix it later when trouble comes up.

i think it's because minimum duration is not explicit in this method signature although it's part of the `EventDetectionCallable` signature